pkg/orchestrator/orchestrator.go

```diff
@@ -1127,6 +1127,10 @@ func (s *ControllerServer) handleCreateOrAttachResource(c *gin.Context) {
 		return
 	}
 
+	if err := s.preCreateOrAttach(c, resourceInfo); err != nil {
+		return
```

Comment:
I'm not returning an error response because at any point an error occures in `preCreateOrAttach`, AbortWithStatusJSON is called to fill in the error message